en/flight_controller/modalai_fc_v1.md

```diff
@@ -1,6 +1,6 @@
 # ModalAI Flight Core v1
 
-The ModalAI Flight Core is a [Pixhawk Series](../flight_controller/pixhawk_series.md) derived flight controller, made in the USA. The Flight Core can be paired with [ModalAI VOXL](https://docs.modalai.com/voxl-datasheet/) for obstacle avoidance and GPS-denied navigation, or used independently as a standalone flight controller.
+The ModalAI Flight Core is a flight controller for PX4, made in the USA. The Flight Core can be paired with [ModalAI VOXL](https://docs.modalai.com/voxl-datasheet/) for obstacle avoidance and GPS-denied navigation, or used independently as a standalone flight controller.
```

Comment:
You might want to change this to NuttX-based flight controller for PX4 to make it clear what sort of architecture this is. I will merge for now :-)